src/actions/audio.ts

```
@@ -11,7 +11,7 @@
 
 import { fileCacheAgentFilter, nijovoiceTextAgentFilter } from "../utils/filters.js";
 import { getAudioArtifactFilePath, getAudioFilePath, getOutputStudioFilePath, resolveDirPath, defaultBGMPath, mkdir, writingMessage } from "../utils/file.js";
-import { localizedText, settings2GraphAIConfig } from "../utils/utils.js";
+import { localizedText, settings2GraphAIConfig, beatId } from "../utils/utils.js";
```

ESLint

[error] Remove this unused import of 'beatId'.

(sonarjs/unused-import)